static/selector.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable quotes */
+/**
+ * Creates the selectors for the chart and updates the chart based on the
+ * selected values.
+ * @param {string} url The url for the application.
+ * @param {string} chartId The id of the chart being drawn.
+ * @param {*} colorScale A d3 colorscale used for the line colors.
```

Comment:
Nit: I'm not sure if we have d3 types available here, but let's use object instead of * since that's at least a little more specific.

Reply:
I can update that, the color scale is a function.